website/src/app/globals.css

```
@@ -410,6 +443,154 @@
+
+/* v2 workflow code-block syntax colors */
+.atx-code .c { color: var(--atx-ink-faint); font-style: italic; }
+.atx-code .k { color: var(--atx-accent); }
+.atx-code .s { color: var(--atx-ok); }
+.atx-code .n { color: var(--atx-info); }
+.atx-code .y { color: var(--atx-warn); }
+
+/* v2 standards strip marquee. Keyframes declared at the top level so
+ * the animation resolves even if Tailwind preflight strips custom keyframes
+ * from other contexts. The component sets `animation` inline as a safety net. */
+@keyframes atxMarqueeScroll {
```

Stylelint (17.7.0)

[error] Expected keyframe name "atxMarqueeScroll" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)